zenoh-protocol/Cargo.toml

```diff
@@ -40,7 +40,7 @@ default = ["zero-copy", "transport_tcp", "transport_udp", "transport_unixsock-st
 
 [dependencies]
 aes-soft = "0.6.4"
-async-std = { version = "=1.8.0", features = ["unstable"] }
+async-std = { version = "=1.9.0", features = ["unstable"] }
```

Comment:
features = ["unstable"] was needed until async-std 1.8.0 to support the async channels. 
Since the channels have been stabilised in async-std 1.9.0, features = ["unstable"] should be hence removed.